### PR TITLE
Sets up React Native for IOS, Android and Web

### PR DESCRIPTION
**Pull request by keving17**
Used create-react-native-app to enable react native on IOS and Android. Added in react-native-web to enable react native on Web as well.

IOS:
22198960_10214077657997925_1673574918_o

Web:
github pull

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "presets": ["babel-preset-expo"],
+  "env": {
+    "development": {
+      "plugins": ["transform-react-jsx-source"]
+    }
+  }
+}
```
Comment (kevinrobinson) [inline .babelrc]: Do we still need this?  Something to tackle after this PR, just wondering.

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+```
+$ npm run android
+```
+to start app in emulator.
+
+To get app running on web, run
+```
+$ npm run web-development
+```
+and a new tab should open with the app already running on it.
```
Comment (kevinrobinson) [inline README.md]: This looks awesome! 👍

File: package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "dependencies": {
+    "clean-webpack-plugin": "^0.1.16",
+    "expo": "^20.0.0",
+    "json-loader": "^0.5.7",
+    "react": "^16.0.0-rc.3",
+    "react-dom": "^16.0.0-rc.3",
+    "react-native": "^0.47.0",
+    "react-native-web": "^0.1.2",
+    "react-native-webpack": "^0.1.1",
+    "webpack": "^3.6.0"
```
Comment (kevinrobinson) [inline package.json]: We should clean these up after, and see if there are dependencies we don't need anymore, but can do this in a separate PR.

File: web/index.web.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+console.log("started")
+// register the app
+AppRegistry.registerComponent('App', () => App);
+
+AppRegistry.runApplication('App', {
+  initialProps: {},
+  rootTag: document.getElementById('root')
+});
+
+console.log("finished")
```
Comment (kevinrobinson) [inline web/index.web.js]: You've got some leftover console.logs here, please remove before merging.

Comment (keving17) [reply web/index.web.js]: sure! done!
